spec/system/page_objects/pages/user_preferences_interface.rb

```diff
@@ -21,8 +21,8 @@ def select_bookmark_after_notification_mode(value)
       end
 
       def save_changes
-        click_button "Save Changes"
-        expect(page).to have_content(I18n.t("js.saved"))
+        click_button(I18n.t("js.save"))
+        find(".saved", exact_text: I18n.t("js.saved"))
```

Comment:
There can be a debate on this, Im not sure this is the best way to do it so far, but we generally have been avoiding expect in page objects. What's your stance on this @tgxworld ?